index.js

```diff
@@ -0,0 +1,34 @@
+const hideimg = document.querySelector('.imgmenu');
+const hamburger = document.querySelector('.mobile--nav');
+const close1 = document.querySelector('.menu-close');
+const item = document.querySelectorAll('.items');
+const newmenu = document.querySelector('#Menu');
+const view1 = document.querySelector('.whoiam');
+const showimg = document.querySelector('.imgmenu1');
+const footimg = document.querySelector('.Menu-foot');
+
+hamburger.addEventListener('click', () => {
+  hideimg.style.display = 'none';
+  showimg.style.display = 'block';
+  newmenu.style.display = 'block';
+  view1.style.display = 'none';
+  hamburger.style.display = 'none';
+  close1.style.display = 'block';
+  footimg.style.display = 'block';
+});
+
+close1.addEventListener('click', () => {
+  view1.style.display = 'block';
+  hamburger.style.display = 'block';
+  hideimg.style.display = 'block';
+  showimg.style.display = 'none';
+  close1.style.display = 'none';
+  newmenu.style.display = 'none';
+});
+
+for (let i = 0; i < item.length; i += 1) {
+  item[i].addEventListener('click', () => {
+    newmenu.style.display = 'none';
+    view1.style.display = 'block';
+  });
+}
```

Comment:
- [ ] When I click one of the items on the menu I get redirected correctly, but then the X is present and not the hamburger menu. Kindly fix this 
https://www.awesomescreenshot.com/video/15393281?key=9d6fa7e058bdccd9568869407f132248

Reply:
- [ ] The menu should extend all the way down and when opened you should be able to see only the menu, also please work on design to make it more similar 

Yours:

![image](https://user-images.githubusercontent.com/62705947/223484603-3470c97e-6125-4510-8990-49b0f99aadc3.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/223484754-672fdd44-9f63-4e2c-966b-978bdbef8b4b.png)